file-uploader/routes/upload_test.go

```diff
@@ -0,0 +1,62 @@
+package routes
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"mime/multipart"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"github.com/ONSdigital/ras-rm-sample/file-uploader/config"
+	"github.com/ONSdigital/ras-rm-sample/file-uploader/file"
+	"github.com/ONSdigital/ras-rm-sample/file-uploader/inject"
+	"github.com/ONSdigital/ras-rm-sample/file-uploader/stub"
+)
+
+var fileProcessorStub file.FileProcessor
+var ctx = context.Background()
+func init() {
+	_, client := stub.CreateTestPubSubServer("testtopic", ctx)
+	fileProcessorStub = file.FileProcessor{
+		Config: config.Config{
+			Port: "8080",
+			Pubsub: config.Pubsub{
+				TopicId: "testtopic",
+				ProjectId: "project",
+			},
+		},
+		Client: client,
+		Ctx: ctx,
+	}
+}
+
+func TestFileUploadSuccess(t *testing.T) {
+	inject.FileProcessor = fileProcessorStub
+	path := "../file/sample_test_file.csv"
+	file, err := os.Open(path)
+	if err != nil {
```

Comment:
Absolutely nothing wrong with this but in test I like to do

assert.Nil(err)

Reply:
I was looking for an assert Nil but goland didnt suggest it. my goland isnt behaving at all, ive biffed it somehow. I'll add it in